models/book.js

```diff
@@ -0,0 +1,57 @@
+//vars used for global scope availability
+var app = app || {};
+// noinspection JSVoidFunctionReturnValueUsed
+app.BookModel = Backbone.Model.extend(
+	{
+		initialize : function (options) {
+		},
+		/**
+		 * Sets url path for API requests
+		 */
+		url: function () {
+			let myURL = document.location.origin + '/wp-json/bookAPI/v1/book';
+			if (this.id) {
+				myURL += '/' + this.id;
+			}
+			return myURL;
+		},
+		/**
+		 * Default values<br>
+		 * post_id is by default the id of the current book instance
+		 */
+		defaults: {
+			post_id : jQuery( '#get_page_id' ).val(),
+			title: '',
+			author: '',
+			genre: '',
+			summary: '',
+		},
+		/**
+		 * Checks if author has 2 words
+		 * @param attrs
+		 * @param options
+		 * @returns {string}
+		 */
+		validate : function (attrs, options) {
+			if (this.get( 'author' ).split( ' ' ).length < 2) {
+				return "author must be a valid name";
+			}
+		},
+		/**
+		 * AJAX request to delete element from the serialized post meta field<br>
+		 */
+		deleteFromSerialized : function () {
```

Comment:
you don't need to write a ajax call to delete a model, backbone does that for you by calling `destroy` method. Details [here](https://backbonejs.org/#Model-destroy)

Reply:
well there is a problem with that, for delete I need both the model id and the post id. 
I know that the destroy method would trigger a delete request but you can't send additional data.
Is there a way to get the post id without having to make an ajax call?